### DISCUSSION TITLE
Isn't it possible to run the style validator using Spectacles CLI?

### DISCUSSION THREAD

**Discussion by adrpino** [spectacles-ci/spectacles#721] [General]
I was looking at the new [style validation](https://docs.spectacles.dev/app/reference/style-validator-rules/) available at Spectacles. I thought that this feature would be available also when using the open source version (using the [CLI](https://docs.spectacles.dev/cli/tutorials/getting-started/)), however I don't manage to find a way to get this done.

Is this feature only available for Cloud (paid) users of Spectacles?

**Comment by DylanBaker** [comment]
Hi @adrpino. Yeah, the linter is only a paid freature of the hosted Spectacles application. It is not available via the CLI.
